packages/core/src/types.ts

```diff
@@ -264,7 +264,7 @@ export type InvokeCreator<
 ) =>
   | PromiseLike<TFinalContext>
   | StateMachine<TFinalContext, any, any>
-  | Subscribable<any>
+  | Subscribable<EventObject>
```

Comment:
This makes sense since we expect that emitted items are of this type: https://github.com/davidkpiano/xstate/blob/14cdef612df29b9e067a34ca96b0750e729c9aff/packages/core/src/interpreter.ts#L1156

Technically we also support strings and SCXML events here at runtime but I don't think that any of those are worth supporting here.

Reply:
I would also be happy with a union type `Subscribable<string | EventObject | SCXML.Event<EventObject>` (not entirely sure of that last part, since I haven't worked with SCXML yet).

Reply:
I think this can be skipped for now and we can just reconsider doing this in the future